spec/models/comment_spec.rb

```diff
@@ -3,6 +3,17 @@
 describe Comment do
   it_behaves_like "email_confirmable"
 
+  describe "#confirm!" do
+    let(:comment) do
+      VCR.use_cassette('planningalerts') { build(:comment, confirmed: true) }
+    end
+
+    it "should run callback once already confirmed" do
```

Comment:
I find this description confusing. Is there a typo? Could it be that the test describes something not happening, but the description says something should happen?